Added Components

Sureshvarthya12 opened the pull request.

Sureshvarthya12 commented:
Added Login page and other components

lokesh234 commented on src/Components/SignUp.jsx:
SignUp should should be a page instead of a component

lokesh234 commented on src/Components/Login.jsx:
Login should be a page instead of a component

lokesh234 commented on src/Components/ForgotPassword.jsx:
ForgotPassword should be a page instead of a component

lokesh234 reviewed (changes requested):
You should generally put the title of the PR as the Story number and title. You should probably create another folder for the pages

Sureshvarthya12 commented:
FE 001 - Create a Sign Up Page

Added authentication pages (Login, Signup, Forgot Password) and Dashboard as pages 

- Created pages directory
- Implemented Login page with mock authentication
- Added Dashboard with user profile and trips tabs
- Implemented navigation between pages

- Tested login flow with correct and incorrect credentials
- Verified dashboard displays correctly after login
- Tested logout functionality
